tests/scale/results/1.1.0/1.1.0.md

```diff
@@ -0,0 +1,224 @@
+# Results for v1.1.0
+
+<!-- TOC -->
+- [Results for v1.1.0](#results-for-v110)
+  - [Summary](#summary)
+  - [Versions](#versions)
+  - [Tests](#tests)
+    - [Scale Listeners](#scale-listeners)
+    - [Scale HTTPS Listeners](#scale-https-listeners)
+    - [Scale HTTPRoutes](#scale-httproutes)
+    - [Scale Upstream Servers](#scale-upstream-servers)
+    - [Scale HTTP Matches](#scale-http-matches)
+  - [Future Improvements](#future-improvements)
+<!-- TOC -->
+
+## Summary
+
+- Memory usage has increased since 1.0
```

Comment:
Significantly?

Reply:
I deployed both 1.0.0 and edge versions in a cluster, and without any additional resources (like HTTPRoutes), 1.0.0 is 17Mb, edge is 16Mb

Reply:
I think we should use the longevity test to see if it's a leak or not. If not, we should definitely look into it, but I don't think I'd block the release over it. But likely a story to track performance issues introduced between 1.0 and 1.1.